infrastructure/src/main/kotlin/internal/detekt/FindChangedFilesTask.kt

```diff
@@ -0,0 +1,89 @@
+package com.redmadrobot.build.internal.detekt
+
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.diff.DiffEntry
+import org.eclipse.jgit.lib.Repository
+import org.eclipse.jgit.storage.file.FileRepositoryBuilder
+import org.eclipse.jgit.treewalk.AbstractTreeIterator
+import org.eclipse.jgit.treewalk.CanonicalTreeParser
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.ConfigurableFileCollection
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+import java.io.Serializable
+import javax.inject.Inject
+
+internal abstract class FindChangedFilesTask @Inject constructor(
```

Comment:
I think the word `Git` should be mentioned in the task name.  
Maybe `CollectGitDiffFilesTask`?

Reply:
Yes, you suggestion is good!